pkgs/development/tools/vagrant/default.nix

```diff
@@ -1,7 +1,10 @@
 { stdenv, lib, fetchurl, buildRubyGem, bundlerEnv, ruby, libarchive
-, libguestfs, qemu, writeText, withLibvirt ? stdenv.isLinux
+, libguestfs, qemu, writeText, withLibvirt ? stdenv.isLinux, withAws ? false
 }:
 
+# dependencies of these two plugins conflict
+assert !(withLibvirt && withAws);
```

Comment:
It is still the case in 2024 and what command did you use to generate the gemset ? Also use throwIf instead of assert